build.gradle

```diff
@@ -27,8 +27,8 @@ dependencies {
     implementation 'info.picocli:picocli:4.5.1'
     implementation 'commons-io:commons-io:2.4'
     implementation "org.gradle:gradle-tooling-api:6.6"
-//    implementation "ch.qos.logback:logback-core:1.2.3"
-//    implementation "ch.qos.logback:logback-classic:1.2.3"
+    implementation "org.apache.logging.log4j:log4j-api:2.13.3"
+    implementation "org.apache.logging.log4j:log4j-core:2.13.3"
```

Comment:
I would move away from log4j , and use logback
we should try to use a logging facade-implementation combination , that would mean sl4j and logback ,
Can this be changed

Reply:
Sure